<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="ToolList">
      <item>
       <property name="text">
        <string>Test0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test8</string>
       </property>
      </item>
     </widget>
     <widget class="QStackedWidget" name="ToolsStackedWidget">
      <widget class="QWidget" name="page_0"/>
      <widget class="QWidget" name="page_1"/>
      <widget class="QWidget" name="page_2"/>
      <widget class="QWidget" name="page_3"/>
      <widget class="QWidget" name="page_4"/>
      <widget class="QWidget" name="page_5"/>
      <widget class="QWidget" name="page_6"/>
      <widget class="QWidget" name="page_7"/>
      <widget class="QWidget" name="page_8"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
